src/Lean/Elab/Inductive.lean

```diff
@@ -25,6 +25,11 @@ open Meta
 builtin_initialize
   registerTraceClass `Elab.inductive
 
+register_builtin_option autoPromoteIndices : Bool := {
```

Comment:
```suggestion
register_builtin_option inductive.autoPromoteIndices : Bool := {
```
seems like it should be scoped in some way, although I'm really missing that option naming style guide :sob:

Reply:
I agree. Fixed in the relevant PR #3590.